### DISCUSSION TITLE
Guide ani-cli stars widget

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#698] [General] [answered]
I kinda want to have a widget showing the stars of ani-cli in my status bar.
Here is my first prototype of a fetching command: 
`curl -s "https://api.github.com/repos/pystardust/ani-cli" | grep stargazers_count | cut -d : -f 2 | tr -d " " | tr -d ","`

You could theoretically hardcode a character range after the grep.
Optimizations welcome

**Comment by CoolnsX** [comment]
Also u can put same tr command to reduce piping...
```tr -d " |,"```

**Comment by CoolnsX** [comment, marked as answer]
Also a better command..
```curl -s "https://api.github.com/repos/pystardust/ani-cli" | sed -nE 's/.*"stargazers_count": (.*),/\1/p'```

**Reply by justchokingaround** [reply]
👑
